### DISCUSSION TITLE
When to shows the total plots with the harvesters on the main machine

### DISCUSSION THREAD

**Discussion by sharyuke** [Chia-Network/chia-blockchain#5103] [Support]
![image](https://user-images.githubusercontent.com/10694303/118363897-04e36f00-b5c9-11eb-8c8e-1fadede576f3.png)

I want to know how many plots is farming , and whether  it is working correctly. 
And why this harvester has not any data transform? I have checked it , it has attempted proof in the list with this harvester.
![image](https://user-images.githubusercontent.com/10694303/118370469-6a8c2700-b5da-11eb-9e83-6ba05316863b.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
